samples/algorithms/HiddenShift.qs

```diff
@@ -0,0 +1,178 @@
+/// # Sample
+/// Hidden shift
+///
+/// # Description
+/// There is a family of problems known as hidden shift problems, in which one
+/// is given two Boolean functions 𝑓 and 𝑔 with the promise that they satisfy
+/// the relation
+///     𝑔(𝑥) = 𝑓(𝑥 ⊕ 𝑠) for all 𝑥
+/// where 𝑠 is a hidden bit string that we would like to find.
```

Comment:
It might, but semantically backticked stuff is usually for code symbols and in this case the s symbol is not a code symbol so it is at least consistent. I'm not too concerned about it.